backend/danswer/connectors/confluence/connector.py

```diff
@@ -200,6 +209,7 @@ class ConfluenceConnector(LoadConnector, PollConnector):
     def __init__(
         self,
         wiki_page_url: str,
+        index_origin: bool = False,
```

Comment:
nit: prefer True here as well, that feels like more of the intuitive, expected behavior (e.g. if you specify a page, that it would get indexed)